src/Tasks.php

```diff
@@ -195,11 +195,11 @@ function configure($opts = [
   /**
    * Perform git checkout of host files.
    */
```

Comment:
Let's add a comment about what this variable is.

Reply:
Yeah and maybe change the actual comment to be accurate while we're there ;)

Reply:
Updated comment, let me know if this needs more work.